README.md

```diff
@@ -1,110 +1 @@
-
-Quickstart:
-
-```bash
-pip install nameguard
-```
-
-```python
-from nameguard import NameGuard
-ng = NameGuard()
-await ng.inspect_name(network_name='mainnet', name='nick.eth')
-```
-
-See the [NameGuard Python README](./api/README.md) for more details.
-
-### Running your own NameGuard instance
-
-See the [NameGuard Python README](./api/README.md) for more details.
-
-## NameGuard Specification
-
-### Checks
-
-1. **Impersonation**: Detects names that could be trying to impersonate a different name by using similar characters. Example: [`vitalìk.eth`](https://nameguard.io/inspect/vitalìk.eth)
-
-2. **Confusables**: Detects characters that can be confused with other characters. Example: [`vitalìk.eth`](https://nameguard.io/inspect/vitalìk.eth)
-
-3. **Font Support**: Checks if the characters in the name are supported by commonly used fonts. Example: [`🛈.eth`](https://nameguard.io/inspect/🛈.eth)
-
-4. **Invisibles**: Detects invisible characters. Example: [`888‍‍.eth`](https://nameguard.io/inspect/888‍‍.eth)
-
-5. **Typing Difficulty**: Detects names that are difficult to type on some keyboards. Example: [`żółć.eth`](https://nameguard.io/inspect/żółć.eth)
-
-6. **Mixed Scripts**: Detects names that contain characters from multiple scripts or alphabets. Example: [`あア.eth`](https://www.nameguard.io/inspect/あア.eth)
-
-7. **Name Wrapper**: Checks if the name is supported by the new ENS Name Wrapper. Example: [`abcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcd.eth`](https://nameguard.io/inspect/abcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcd.eth)
-
-8. **ENSIP-15**: Checks if the name is normalized according to ENSIP-15. Example: [`bitсoin.eth`](https://nameguard.io/inspect/bitсoin.eth)
-
-9. **Punycode**: Checks if the name is compatible with Punycode encoding. Example: [`ab--abc.eth`](https://www.nameguard.io/inspect/ab--abc.eth)
-
-10. **Unknown Labels**: Checks if the name contains unknown labels. Example: [`[5bc926fc40cc7c49e0df6dddf26e4dc7b9d6d32f4a55d4f0670320dbf414afd2].byongdok.eth`](https://nameguard.io/inspect/[5bc926fc40cc7c49e0df6dddf26e4dc7b9d6d32f4a55d4f0670320dbf414afd2].byongdok.eth)
-
-11. **Decentralized Name**: Checks if the name is decentralized (unruggable). Example: [`example.com`](https://www.nameguard.io/inspect/example.com)
-
-12. **NameWrapper fuses**: Checks that the NameWrapper configuration of a name is safe.
-
-## License
-
-Licensed under the MIT License, Copyright © 2023-present [NameHash Labs](https://namehashlabs.org).
-
-See [LICENSE](./LICENSE) for more information.
+# Namekit
```

Comment:
@notrab As a temporary step in the new root-level NameKit readme, could you please include a link to the retained / renamed "nameguard.md"? The goal of this suggestion is to retain some better discoverability for where to learn more about getting started with NameGuard. Appreciate your advice.

Reply:
I planned to have a list of all the packages, internal packages and apps with links to their folders. I'll do that probably later today.

Reply:
@notrab Ok sounds awesome :+1:

I made a temporary solution in this commit: https://github.com/namehash/nameguard/pull/260/commits/9514aa7bac9474a78d2288e971621077013d2f26

Reply:
Awesome, thank you @lightwalker-eth